### DISCUSSION TITLE
upon startup only ekg-capture is available

### DISCUSSION THREAD

**Discussion by gour** [ahyatt/ekg#117] [Q&A] [answered]
Hello,

I'm confused with the fact that after Emacs startup, only `ekg-capture` function is available,
iow. it looks as that I've to capture something **before** being able to browse the notes in the database. Is it expected?

**Comment by ahyatt** [comment, marked as answer]
Yes, thanks for pointing this out.  I need to add autoloads so the main functions are all available - but once I've added them, I can't remove them in anymore, so I've been avoiding it for a while.  But probably ekg is mature enough that the main functions won't change, so I should just go ahead and do that.  I'll fix this soon.

**Reply by ahyatt** [reply]
Yesterday I created a new release (0.4.3) that should solve this problem; please let me know if you still experience this issue after using the new release.

**Comment by gour** [comment]
> Yesterday I created a new release (0.4.3) that should solve this problem; please let me know if you still experience this issue after using the new release.

I've updated the package and so far everything is fine.

Thank you very much!
